packages/protocol/thread/thread.ts

```diff
@@ -455,23 +435,23 @@ class _ThreadFront {
     const abilities = await this.recordingCapabilitiesWaiter.promise;
     const { result } = frameId
       ? await client.Pause.evaluateInFrame(
-          {
-            frameId,
-            expression: text,
-            useOriginalScopes: true,
-            pure: abilities.supportsPureEvaluation && pure,
-          },
-          this.sessionId!,
-          pauseId
-        )
+        {
+          frameId,
+          expression: text,
+          useOriginalScopes: true,
+          pure: abilities.supportsPureEvaluation && pure,
+        },
+        this.sessionId!,
+        pauseId
+      )
       : await client.Pause.evaluateInGlobal(
-          {
-            expression: text,
-            pure: abilities.supportsPureEvaluation && pure,
-          },
-          this.sessionId!,
-          pauseId
-        );
+        {
+          expression: text,
+          pure: abilities.supportsPureEvaluation && pure,
+        },
+        this.sessionId!,
+        pauseId
+      );
```

Comment:
My `prettier` made these indent changes. Is it wrong or should I leave them in?

Reply:
~~Eh, if it passes lint let's leave it.~~

well lint failed :)  double-check settings I guess?

Reply:
Ah, shoot. Well, I'm trying out lazy.vim and it has some different lsp configs than I usually use 🤔